src/components/Wrappers/withDeclareLost.js

```diff
@@ -47,8 +47,8 @@ const withDeclareLost = WrappedComponent => class WithDeclareLost extends React.
       <>
         <WrappedComponent
           declareLost={this.declareLost}
-          declarationInProgress={declarationInProgress}
-          toggleButton={this.toggleButton}
+          declareLostInProgress={declareLostInProgress}
+          setDeclareLostInProgress={this.setDeclareLostInProgress}
```

Comment:
We can remove these props just like in #2289, right? i.e. they're passed directly to `<LoanActionDialog>` via props below, so there's no need to propagate them through the wrapper.

Reply:
Hello @zburke in current case we cannot remove them because we use them in stripes/ui-users/src/views/LoanDetails/LoanDetails.js